.github/workflows/maven-release.yml

```diff
@@ -42,16 +42,16 @@ jobs:
           git_tag_gpgsign: true
           git_push_gpgsign: false
     - name: Cache local Maven repository
-      uses: actions/cache@v2
+      uses: actions/cache@v3
       with:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-maven-
     - name: Prepare Release
       id: release
       run: >
-          mvn -V -gs $GITHUB_WORKSPACE/generated-settings/settings.xml -B -P sonatype-staging,release -e release:clean release:prepare && 
-          echo "::set-output name=RELEASED_VERSION::$(grep scm.tag= release.properties | cut -d'=' -f2 | cut -c2-)"
+          mvn -V -gs $GITHUB_WORKSPACE/generated-settings/settings.xml -B -P sonatype-staging,release -e release:clean release:prepare &&
+          echo "RELEASED_VERSION=G$(grep scm.tag= release.properties | cut -d'=' -f2 | cut -c2-)" >> $GITHUB_OUTPUT
```

Comment:
Please see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

also, added `G` here because there's a bug to generate release tag that used to be `"rammaticus-1.2.5"` (cutting first letter of `grammaticus-1.2.5`, but forgot appending `G` to capitalize) otherwise, we should remove `cut - c2-` part and create tag as `grammaticus-1.2.5`.  either way is fine to me...